Tests/FoundationEssentialsTests/BufferViewTests.swift

```diff
@@ -19,17 +19,16 @@ import TestSupport
 #endif
 
 final class BufferViewTests: XCTestCase {
+    func testAll() async {
+        await XCTestScaffold.runAllTests(hostedBy: self)
+    }
```

Comment:
This should be in it's own file so other `swift-testing` tests can use it as well.